### PR TITLE
chore: update version to 4.1.28

### PR DESCRIPTION
**Pull request by Aarebecca**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#### 4.1.28 (2021-09-23)
+
```
Comment (hustcc) [inline CHANGELOG.md:2]: changlog

**Comment by hustcc** [comment]
本地运行一下 tnpm run ci

**Comment by Aarebecca** [comment]
> 本地运行一下 tnpm run ci

运行了，本地ci过了

File: CHANGELOG.md (hunk 1 above)
Comment (Aarebecca) [reply CHANGELOG.md:2]: 欸，为什么是空的

**Pull request closed without merging**